docs/_config.yml

```diff
@@ -1 +0,0 @@
-theme: jekyll-theme-hacker
```

Comment:
This one cannot be deleted as it is the theme I selected this AM

Reply:
Added it back